### ISSUE TITLE
VDSL3 bugs

### ISSUE THREAD

**Issue by rcannood** [viash-io/viash#139]
* `meta[resources_dir]` should contain an absolute path, not a relative path
* `'''` is incorrectly escaped to `\\'\\'\\'` inside the main.nf
* argument `{ name: --foo, type: file, multiple: true }` should not create a `VIASH_PAR_FOO` variable when `--foo` is not passed

**Comment by rcannood** [comment]
Fixed in Viash 0.5.13

**Comment by rcannood** [comment]
New bug: 

```groovy

    | groupTuple()

    // Step 4: convert to h5ad
    | map { id, input -> [ id, input, params ]}
```

This will not work because an array of files is not detected.

On the other hand, `    | map { id, input -> [ id, [input: input], params ]}` does work.

**Comment by rcannood** [comment]
Also solved in 0.5.13!
